### PR TITLE
Enhancement - Updating all EWSType_ResponseMessageType to extend the base class

### PR DESCRIPTION
**Pull request by jstruzik**
I noticed when attempting to handle `EWSType_ResponseMessageType`s that I wanted to be able to retrieve the `MessageText` of a potentially failed response. This would have been a huge PITA without making things a little more generic.

This PR correctly extends all of the `ResponseMessageType` classes. I'll probably make a PR for the main `php-ews` library for quality of life.

### CODE REVIEW

File: EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php [hunk 1]
```diff
@@ -32,43 +19,4 @@ class EWSType_FindMailboxStatisticsByKeywordsResponseMessageType extends EWSType
      * @var EWSType_MailboxStatisticsSearchResultType
      */
     public $MailboxStatisticsSearchResult;
-
-    /**
-     * Provides a text description of the status of the response.
-     *
-     * @since Exchange 2013
```
Comment (Rican7) [inline EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php:39]: Hmm, only thing worrying me about this is that these properties don't seem to have a consistent minimum version requirement across all response types. Not sure how we can document or handle that properly, in this case. :/

**Comment by Rican7** [comment]
Everything else looks good. :smiley:

File: EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php (hunk 1 above)
Comment (jstruzik) [inline EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php:39]: I'm not really sure if these `@since` tags are accurate. Unless it's for the entire class itself.

Comment (Rican7) [inline EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php:39]: We'd have to cross-reference the SOAP API documentation. There's no reason why properties can't be added between versions regardless of the class, though.

Comment (jstruzik) [inline EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php:39]: I mean the `@since` tag doesn't really enforce any restriction based on the version of exchange the WSDL or SOAP client you're using. Either way the object is going to be exactly the same structurally.

You'd almost have to make a separate class each time a new property is introduced right? Or use some ridiculousness with annotations.

Comment (Rican7) [inline EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php:39]: Yea, absolutely. I'm just saying that now the `@since` doc tags for `EWSType_ResponseMessageType` are invalid. Maybe they should just be removed? Maybe with a special note regarding the polymorphic extension that has a conditional inclusion date?

Comment (jstruzik) [inline EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php:39]: Yeah I could do something like that in a separate PR that gets merged in before this one.

Comment (Rican7) [inline EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php:39]: Wouldn't a separate PR lose its context of the removal, though? That would make the removal seem malicious or without point. Right?

Comment (jstruzik) [inline EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php:39]: But we'd remove it repo-wide.

Comment (Rican7) [inline EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php:39]: Oh, nooooo, I was just saying to remove the `@since` tags from the `EWSType_ResponseMessageType` since they'd no longer be accurate and wouldn't make sense as a base-class. The others are fine and quite helpful.

Comment (jstruzik) [inline EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php:39]: Ah but of course!

:tophat: 
:smile:

**Comment by brianmuse** [comment]
:+1: LGTM aside from things already said.

**Comment by jstruzik** [comment]
This PR has been updated to only extend the base class and not to delete the class properties as they each may hold valuable information in the comments.

**Comment by Rican7** [comment]
Makes sense to me :+1:

**Comment by brianmuse** [comment]
:+1:
